lightgbm/src/main/scala/com/microsoft/azure/synapse/ml/lightgbm/swig/SwigUtils.scala

```diff
@@ -31,8 +31,8 @@ abstract class ChunkedArray[T]() {
   def add(value: T): Unit
```

Comment:
Changes were made in this class because 1) we got build warnings about hidden classes due to casing, and 2) the new reflection code was confused somehow due to filenames that only differed in casing.  Corrected the casing to fix those and make a proper companion class as intended.